trial-config-service/src/main/java/uk/ac/ox/ndph/mts/trialconfigservice/TrialConfigServiceApplication.java

```diff
@@ -0,0 +1,15 @@
+package uk.ac.ox.ndph.mts.trialconfigservice;
```

Comment:
on the other services (sample/practitioner) we had a pattern of trial_config_service which is similar to the maven artifact id defined on the pom.
Not sure what is correct, but we should decide on 1 convention

Reply:
Renamed to trial_config_service. We will now use this convention